documentation/latest/gsg/ocp.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: documentation
+title: Run flotta on OpenShift
+---
+
+Flotta can run on top of any Kubernetes distributions, also, main usage in
+production is on top of OpenShift cluster. 
+
+To install OpenShift, the best way is to follow [OpenShift Assisted
+installer](https://cloud.redhat.com/blog/how-to-use-the-openshift-assisted-installer).
+
+Check OpenShift cluster status first:
+
+```shell
+$ -> oc status
+In project default on server https://api.mycluster.redhat.com:6443
+
+svc/openshift - kubernetes.default.svc.cluster.local
+svc/kubernetes - 10.128.0.1:443 -> 6443
+
+View details with 'oc describe <resource>/<name>' or list resources with 'oc get all'.
+```
+
+Flotta operator has a few tools that helps to install flotta, for that, let's
+clone it:
+
+```shell
+git clone https://github.com/project-flotta/flotta-operator -b main --depth 1
+cd flotta-operator
+```
+
+Now let's install Flotta on the cluster:
+
+```shell
+make deploy IMG=quay.io/project-flotta/flotta-operator:latest TARGET=ocp
+```
+
+A bunch of CRDs are now created, where the definitions can be found
+[here](../operations/crd.md):
+
+```shell
+$ -> oc  api-resources --api-group="management.project-flotta.io"
+NAME                      SHORTNAMES   APIVERSION                              NAMESPACED   KIND
+edgeconfigs                            management.project-flotta.io/v1alpha1   true         EdgeConfig
+edgedevices                            management.project-flotta.io/v1alpha1   true         EdgeDevice
+edgedevicesets                         management.project-flotta.io/v1alpha1   true         EdgeDeviceSet
+edgedevicesignedrequest   edsr         management.project-flotta.io/v1alpha1   true         EdgeDeviceSignedRequest
+edgeworkloads                          management.project-flotta.io/v1alpha1   true         EdgeWorkload
+playbookexecutions                     management.project-flotta.io/v1alpha1   true         PlaybookExecution
+```
+
+At the same time, in the flotta namespace an operator should be running:
```

Comment:
This is for the latest so should we update that with edge-api pod is running as well?